src/libasr/ASR.asdl

```diff
@@ -277,6 +277,8 @@ expr
     | StringOrd(expr arg, ttype type, expr? value)
     | StringChr(expr arg, ttype type, expr? value)
 
+    | CPtrCompare(expr left, cmpop op, expr right, ttype type, expr? value)
```

Comment:
The `cmpop` has various things like `>`. What does that mean for `CPtr`? I think we only want `==` and `!=`.

Reply:
C has pointer comparisons for different `cmpop`  as seen [here](https://stackoverflow.com/questions/9086372/how-to-compare-pointers).

Reply:
I see. We can allow it in ASR, since eventually we want to have a C frontend too. But in LPython I would mainly support `==` and `!=` for now for CPtr. The other ones we can, but don't have to right now.